src/wallet/puzzles/p2_delegated_puzzle_or_hidden_puzzle.clvm.MANUALLY_COMPILED

```diff
@@ -93,7 +93,7 @@
 
 ; the hex file disassembles to:
```

Comment:
Maybe

`; the hex file disassembles to the equivalent of`

Reply:
I like
`; the hex file disassembles to the equivalent of`

Could also be something like
`; the hex file was compiled from / the following code was the input`

P.S. If we keep this puzzle as clvm (not ChiaLisp) source for a while, we might want to move it to a separate file, and just assemble it, until we move back to the ChiaLisp one.

I think it's fine to have some compiled ChiaLisp, and some assembly source files.

Reply:
I got the `(q . 0)` => `0` compiler optimization working and this file now seems to compile to exactly that output, so I will remove the comment soon.